spec/patch_ruby_spec.rb

```diff
@@ -0,0 +1,18 @@
+describe Patch do
+  context 'Models' do
+    it 'defines all models' do
+      constants.each do |constant|
+        expect { Patch.const_get(constant) }.not_to raise_error
+      end
+    end
+  end
+
+  def constants
+    # Given a file path return the constant of that path, for example:
```

Comment:
All recommendations on how to achieve this in a smoother way is welcome. I'm sure there's a more straightforward approach to this that I can't think of right now.

Reply:
That looks pretty OK for now! Let's do a survey of other SDKs a bit later for best practices?